frontend/src/components/common/range-selector/range-selector.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { FC, useEffect, useState } from 'react';
+
+import { FiltersNames } from '@common/enums/cars/filters-names.enum';
+import { RangeValueType } from '@common/types/cars/range-item.type';
+import { SelectField } from '@components/common/select-field/select-field';
+import { MenuItem, SelectChangeEvent } from '@mui/material';
+
+type Props = {
+  list: Array<string>;
+  minTitle: string;
+  maxTitle: string;
+  minFilterName: FiltersNames;
+  maxFilterName: FiltersNames;
+  selectedMin: string;
+  selectedMax: string;
+  onChange: (range: RangeValueType[]) => void;
+};
+
+const RangeSelector: FC<Props> = (props) => {
+  const [range, setRange] = useState({
+    min: props.selectedMin,
+    max: props.selectedMax,
+  });
```

Comment:
I believe it would be easier to have 2 separate `useState` for the `min` and `max`. Then there would be no mutation issue and the values update would look slimmer

Reply:
Yes, code is now more readable and understandable, thank you!